docs/source/index.md

```diff
@@ -20,113 +20,183 @@ ExecuTorch provides support for:
 
 * **Strong Model Support** LLMs (Large Language Models),
   CV (Computer Vision), ASR (Automatic Speech Recognition), TTS (Text To Speech)
-* **All Major Platforms** Android, Mac, Linux, Windows
+* **All Major Platforms** Android, iOS, macOS, Linux, Windows, Embedded
 * **Rich Acceleration Support** Apple, Arm, Cadence, MediaTek, NXP, OpenVino, Qualcomm, Vulkan, XNNPACK
 
 ### Documentation Navigation
-#### Introduction
+
+#### Getting Started
```

Comment:
I don't like this is called "Getting Started" 

Can we change to something like "Background", "Introduction", "Design and Architecture"?